src/xcode/ENA/ENA/Source/Workers/__tests__/StoreTests.swift

```diff
@@ -0,0 +1,119 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import XCTest
+@testable import ENA
+
+final class StoreTests: XCTestCase {
+	private var store: SecureStore!
+
+	override func setUp() {
+		store = SecureStore(at: URL(staticString: ":memory:"), key: "123456")
+	}
+
+	func testResultReceivedTimeStamp_Success() {
+		XCTAssertNil(store.testResultReceivedTimeStamp)
+		store.testResultReceivedTimeStamp = Int64.max
+		XCTAssertEqual(store.testResultReceivedTimeStamp, Int64.max)
+		store.testResultReceivedTimeStamp = Int64.min
+		XCTAssertEqual(store.testResultReceivedTimeStamp, Int64.min)
+	}
+
+	func lastSuccessfulSubmitDiagnosisKeyTimestamp_Success() {
```

Comment:
Xcode doesn't seem to recognize functions in Unit Tests without a prefix of 'test'. This applies to all following functions. 

If this was on purpose feel free to resolve this conversation

Reply:
Fixed